app/homepage.py

```diff
@@ -416,19 +356,19 @@ def Homepage():
                     html.Div(id='alert_btn_switch_view'),   # Associated with the alert banner in risks mode
 
                     # Simple placeholder - Source of truth for the map style being viewed
-                    html.Div(id='current_map_style', children='alerts'),
-
-                    # Hidden html.Div storing the URL address of the detection frame of the latest alert
-                    html.Div(id='img_url', style={'display': 'none'})],
-                    md=9),
+                    html.Div(id='current_map_style', children='alerts', style={'display': 'none'}),
+                    # Hidden div storing the webscocket message sent by the API
```

Comment:
```suggestion
                    # Hidden div storing the websocket message sent by the API
```

Just a very small typo 😇

Reply:
Thanks!